app/charts/chart-config-ui-options.ts

```diff
@@ -6,7 +6,7 @@ import {
   SortingOrder,
   SortingType,
 } from "@/config-types";
-import { isTemporalDimension } from "@/domain/data";
+import { isTemporalDimension, isTemporalOrdinalDimension } from "@/domain/data";
```

Comment:
Is it to support the inventory time periods from WSL ?

Reply:
Exactly :)